build.gradle

```
@@ -0,0 +1,28 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot' version '2.7.16'
+    id 'io.spring.dependency-management' version '1.0.15.RELEASE'
+}
+
+repositories {
+    mavenCentral()
+}
+
+group = 'hello'
+version = '0.0.1-SNAPSHOT'
+
+java {
+    sourceCompatibility = '11'
+}
+
+dependencies {
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.3'
+    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
+
+    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    testImplementation 'org.mockito:mockito-core:4.8.0'
```

```suggestion
    // JUnit5 의존성
    testImplementation 'org.junit.jupiter:junit-jupiter:5.8.2'

    // Mockito 의존성
    testImplementation 'org.mockito:mockito-core:4.5.1'
    testImplementation 'org.mockito:mockito-junit-jupiter:4.5.1'
    // assertJ 의존성
    testImplementation 'org.assertj:assertj-core:3.22.0'
```

스프링 테스트 의존성에 다 포함되어 있긴한데, 각각의 의존성으로 정의하시는 것이 좋습니다.
불필요한 의존성도 비용입니다.

---

넵!! 감사합니다 [b588f5b](https://github.com/prgrms-be-devcourse/java-library-management/pull/26/commits/b588f5bdca2cae6df3b615d5366c828b90f8e6d9) 과 같이 반영했습니다 🙆🏻‍♀️